lib/tasks_private/vpc.rake

```diff
@@ -0,0 +1,168 @@
+namespace :spec do
+  require 'ibm_cloud_sdk_core'
+  require 'ibm_cloud_resource_controller'
+  require 'ibm_vpc'
+
+  base_dir = File.join(ManageIQ::Providers::IbmCloud::Engine.config.root.to_s, 'spec')
+  cass_dir = File.join(base_dir, 'vcr_cassettes/manageiq/providers/ibm_cloud')
+  spec_dir = File.join(base_dir, 'models/manageiq/providers/ibm_cloud')
+
+  # Prerequisites:
+  # 1. Log in with IBM Cloud CLI i.e. `ibmcloud login -sso`
+  # 2. Place IBM Cloud API key in secrets config
+  desc "Generate refresher VCR"
+  task :refresh_init => :environment do
```

Comment:
@agrare Do we have a naming convention for the task that generates a new env?

Reply:
Azure and openstack both name these differently so we should stick with one and update the rest
`Azure: rake spec:regenerate:all`
`Openstack: rake vcr:rerecord`
Kubernetes just has a shell script

Reply:
Don't like `spec:regenerate:all` because we aren't regenerating all specs haha  `vcr:rerecord` works unless @Fryguy has a better idea